src/media/css/app-list.styl

```diff
@@ -45,7 +45,7 @@ $desktop-tile-padding = $tile-padding-y $tile-padding-x;
         position: absolute;
         width: 100%;
     }
-    &.previews-expanded .app-list-app {
+    &.previews-expanded .app-list-app:not(.app-list-website) {
```

Comment:
Probably makes more sense. Would be good to split out the app only styles but it's definitely more work for now.